test/jdk/java/foreign/TestLinker.java

```diff
@@ -250,7 +251,13 @@ public void sequenceOfZeroElements() {
         var padding5a1 = MemoryLayout.paddingLayout(5);
         var struct8a8 = MemoryLayout.structLayout(sequence0a8, sequence3a1, padding5a1);
         var fd = FunctionDescriptor.of(struct8a8, struct8a8, struct8a8);
-        linker.downcallHandle(fd);
+        if (linker.getClass().equals(FallbackLinker.class)) {
```

Comment:
Alternatively, we could move this test case to a separate file (perhaps with other test cases that test empty aggregates), and then exclude it for the fallback linker using `@requires jdk.foreign.linker != "FALLBACK"`.

I don't mind this approach though.